projects/canopy/src/lib/details/details.component.ts

```diff
@@ -32,6 +35,46 @@ export class LgDetailsComponent implements AfterContentInit, OnDestroy {
   panelHeading: LgDetailsPanelHeadingComponent;
 
   @Input() isActive = false;
+
+  _showIcon = true;
+  @Input()
+  set showIcon(showIcon: boolean) {
+    this._showIcon = showIcon;
+    if (this.panelHeading) {
+      this.panelHeading.showIcon = showIcon;
+      this.panelHeading.cdr.detectChanges();
```

Comment:
I'm just wondering if this could be done in the panel heading setter for showIcon, then you wouldn't need to expose the ChangeDetectorRef.

Reply:
Ah yes, nice one Andy. I've updated that now, no longer exposing `ChangeDetectorRef` 👍